addons/web_editor/models/html_field_history_mixin.py

```diff
@@ -141,4 +140,21 @@ def html_field_history_get_comparison_at_revision(self, field_name, revision_id)
             field_name, revision_id
         )
 
-        return generate_comparison(self[field_name] or "", restored_content)
+        return generate_comparison(restored_content, self[field_name] or "")
+
+    def html_field_history_get_unified_diff_at_revision(self, field_name, revision_id):
```

Comment:
Wait, why change web_editor in master? The whole module is going to be removed before the OXP. If the field history mixin is only in web_editor right now, it needs to be moved to html_editor ASAP.

Reply:
yes indeed it should be moved. But we should do it in another PR right ?